README.rst

```diff
@@ -22,12 +22,22 @@ You can see the full rendered docs at: http://stestr.readthedocs.io/en/latest/
 Overview
 --------
 
-stestr is a fork of the `testrepository`_ that concentrates on being a
-dedicated test runner for python projects. The generic abstraction
-layers which enabled testr to work with any subunit emitting runner are gone.
-stestr hard codes python-subunit-isms into how it works. The code base is also
-designed to try and be explicit, and to provide a python api that is documented
-and has examples.
+stestr is parallel Python test runner designed to execute `unittest`_ test
```

Comment:
```suggestion
Stestr is parallel Python test runner designed to execute `unittest`_ test
```
or `stestr`?

Reply:
I've been treating `stestr` as a [proper noun](https://en.wikipedia.org/wiki/Proper_noun) (being the name of the project) with the lowercase 's'. It kinda looks weird to me with the uppercase 'S'. That being said I don't really feel super strongly about it, so if you want to capitalize it everywhere we can do that.